Fix broken tabbed code blocks. Minor cleanup.

dwightjl opened the pull request:
- Convert tabbed code blocks to Markdown format. Markdown ignores tabbed code blocks in this format. See https://docs.chain.link/docs/running-a-chainlink-node/
- Trim unnecessary extra spaces at ends of lines
- Add newline at EOF

StephenFluin reviewed (changes requested):
Thanks so much for identifying that these pages were broken!

We're trying to slowly move away from readme MD towards more standard MD. Can you use this format instead for the tabs:

    ```shell Amazon Linux 2
    sudo amazon-linux-extras install -y docker
    sudo systemctl start docker
    sudo gpasswd -a $USER docker
    exit
    # log in again
    ```
    ```shell CentOS
    curl -sSL https://get.docker.com/ | sh
    sudo systemctl start docker
    sudo usermod -aG docker $USER
    exit 
    # log in again
    ```

dwightjl commented:
Awesome. I didn't know MD supported tabbed codeblocks that way.

Updated and squashed the commits together. This looks correct on my staging server.
